python_notebooks/1kg_rna_quantification_service.ipynb

```diff
@@ -336,39 +353,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": 13,
    "metadata": {
     "collapsed": false
    },
    "outputs": [
     {
-     "name": "stdout",
+     "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Expression Level: ENST00000234590.8\n",
-      " id: WyIxa2dlbm9tZXMiLCIxa2ctcm5hIiwiaGc5NiIsIkVOU1QwMDAwMDIzNDU5MC44Il0\n",
-      " expression: 2805.25 TPM\n",
-      "\n",
-      "Expression Level: ENST00000374550.7\n",
-      " id: WyIxa2dlbm9tZXMiLCIxa2ctcm5hIiwiaGc5NiIsIkVOU1QwMDAwMDM3NDU1MC43Il0\n",
-      " expression: 3597.17 TPM\n",
-      "\n",
-      "Expression Level: ENST00000396651.7\n",
-      " id: WyIxa2dlbm9tZXMiLCIxa2ctcm5hIiwiaGc5NiIsIkVOU1QwMDAwMDM5NjY1MS43Il0\n",
-      " expression: 2813.83 TPM\n",
-      "\n",
-      "Expression Level: ENST00000370321.7\n",
-      " id: WyIxa2dlbm9tZXMiLCIxa2ctcm5hIiwiaGc5NiIsIkVOU1QwMDAwMDM3MDMyMS43Il0\n",
-      " expression: 2234.57 TPM\n",
-      "\n",
-      "Expression Level: ENST00000368567.4\n",
-      " id: WyIxa2dlbm9tZXMiLCIxa2ctcm5hIiwiaGc5NiIsIkVOU1QwMDAwMDM2ODU2Ny40Il0\n",
-      " expression: 8211.31 TPM\n",
-      "\n",
-      "Expression Level: ENST00000242465.3\n",
-      " id: WyIxa2dlbm9tZXMiLCIxa2ctcm5hIiwiaGc5NiIsIkVOU1QwMDAwMDI0MjQ2NS4zIl0\n",
-      " expression: 1108.49 TPM\n",
-      "\n"
+      "ERROR:ga4gh.client.client:404 {\"errorCode\": 548919146, \"message\": \"RnaQuantificationSet with name 'WyIxa2dlbm9tZXMiLCIxa2ctcm5hIl0' not found\"}\n"
```

Comment:
Should probably store the RnaQuantificationSetId when it is first retrieved and use that variable here